pkgs/development/python-modules/huepy/default.nix

```
@@ -0,0 +1,33 @@
+{
+  lib,
+  fetchPypi,
+  buildPythonPackage,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "huepy";
+  version = "1.2.1";
+  pyproject = true;
+
+  src = fetchPypi {
+    pname = "huepy";
+    inherit version;
+    hash = "sha256-Wym+73lzEvt2BhiLxc2Y94q49+AVdkJ6kxLxybILdZ0=";
+  };
+
+  build-system = [ setuptools ];
+
+  pythonImportsCheck = [ "huepy" ];
+
+  # no test
+  doCheck = false;
+
+  meta = {
+    description = "Print awesomely in terminals";
+    homepage = "https://pypi.org/project/huepy/";
+    license = "licenses.gpl3Only";
```

```suggestion
    license = lib.licenses.gpl3Only;
```
Please use license attribute from `lib.licenses`